vcr/config.go

```diff
@@ -37,6 +37,8 @@ type Config struct {
 type OIDC4VCIConfig struct {
 	// Enabled indicates if issuing and receiving credentials over OIDC4VCI is enabled
 	Enabled bool `koanf:"enabled"`
+	// URL defines the base URL of the OIDC4VCI issuer and wallet
+	URL string `koanf:"url"`
```

Comment:
Will this be any different than the `publicURL`?

Reply:
Yes, since `auth.publicurl` requires a publicly trusted certificate, while this one PKIoverheid Private Services.

I'm not 100% convinced this is the right solution (configuring it) and even if so, we might want to restructure it. But we might also just want to register a service on the vendor's DID document. But this discussion is part of https://github.com/nuts-foundation/nuts-node/issues/2112